module1.js

```diff
@@ -111,6 +130,9 @@ function question7(array1, array2, extraElement) {
 
 function question8(retailPrice, taxAmount) {
   // TODO:
+  var tax;
+  // tax = retailPrice * taxAmount;
+  return retailPrice + retailPrice * taxAmount;
```

Comment:
Looks like a good attempt, but you did not consider that the input is a string and output is also considered a string and you can't use mathematical operators like `+` and `*` with strings